package.json

```
@@ -7,12 +7,19 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
+    "classnames": "^2.3.2",
+    "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.5",
     "react-scripts": "5.0.1",
+    "redux-devtools-extension": "^2.13.9",
     "web-vitals": "^2.1.4"
   },
+  "engines": {
+    "node": "18.14.x",
+    "npm": "9.5.x"
+  },
```

```suggestion
  "engines": {
    "node": "18.x",
    "npm": "9.x"
  },
```
Rumman Bhai, Can we do that? Please check and let me know.